### ISSUE TITLE
Actual petition?

### ISSUE THREAD

**Issue by lroggendorff** [briandailey/newnashville#2]
First: this is awesome.

Second, I might be missing something, but wouldn't it be even more awesomer if there were some way to actually attach names to this manifesto?
- A change.org thing?
- Some other online petition platform?
- A list of names on the readme itself, PR to add yours?

**Comment by briandailey** [comment]
I like the last idea but that eliminates those without a GitHub account.
Also would need a clearer idea of what exactly signatories are expected to do or not do.

**Comment by lroggendorff** [comment]
@briandailey, yep, exactly. Option there is the simplest on this end, but more complicated for many, I'd guess.

As for responsibilities, I hadn't really thought too much, I was more just thinking of it as a way for people to show support. It would of course be best to make any responsibilities clear (either way) in the manifesto itself.
